collections/m2m gateway/eServiceTemplates/Get EService Template.bru

```diff
@@ -0,0 +1,19 @@
+meta {
+  name: Get eService template
```

Comment:
Just for consistency with existing eservice-related calls (same for other Bruno files in this PR)
```suggestion
  name: Get eservice template
```